ldap/servers/plugins/uiduniq/uid.c

```diff
@@ -1392,13 +1392,13 @@ preop_modrdn(Slapi_PBlock *pb)
              */
             if (NULL != markerObjectClass) {
                 /* Subtree defined by location of marker object class */
-                result = findSubtreeAndSearch(slapi_entry_get_sdn(e), attrNames, attr, NULL,
-                                              requiredObjectClass, superior,
+                result = findSubtreeAndSearch(superior, attrNames, attr, NULL,
```

Comment:
Regarding this regression, is it the path that was failing ?

Reply:
Nope, this path was okay but I've decided to clear the things up a bit (so it uses the same logic as searchAllSubtrees, and we have one call less in the execution)